### PR TITLE
code formatting

### PR DESCRIPTION
**Pull request by BillWagner**
Format the code in the interfaces, arrays and structs chapters.

### CODE REVIEW

File: standard/arrays.md [hunk 1]
```diff
@@ -120,10 +124,13 @@ Because of array covariance, assignments to elements of reference type arrays in
 > ```csharp
 > class Test
 > {
->     static void Fill(object[] array, int index, int count, object value) {
+>     static void Fill(object[] array, int index, int count, object value) 
+>     {
 >         for (int i = index; i < index + count; i++) array[i] = value;
 >     }
```
Comment (BillWagner) [inline standard/arrays.md]: I thought about using an expression bodied member here, but it looked wrong to me. What are your thoughts?

File: standard/interfaces.md [hunk 2]
```diff
@@ -360,18 +376,22 @@ For interfaces that are strictly single-inheritance (each interface in the inher
...
+> interface INumber : IInteger, IDouble {}
+> 
 > class C
 > {
->     void Test(INumber n) {
->         n.Add(1); // Invokes IInteger.Add
->         n.Add(1.0); // Only IDouble.Add is applicable
+>     void Test(INumber n)
+>     {
+>         n.Add(1);              // Invokes IInteger.Add
```
Comment (jskeet) [inline standard/interfaces.md]: Remove one space before the comment (unless I've misread somehow)

File: standard/interfaces.md [hunk 3]
```diff
@@ -506,16 +536,18 @@ For purposes of implementing interfaces, a class or struct may declare ***explic
...
-> interface IDictionary<K,V>
+>
+> interface IDictionary<K, V>
 > {
 >     V this[K key];
 >     void Add(K key, V value);
 > }
-> class List<T>: IList<T>, IDictionary<int,T>
+>
+> class List<T> : IList<T>, IDictionary<int,T>
```
Comment (jskeet) [inline standard/interfaces.md]: Add space after `int`

**Review by jskeet — APPROVED** [review]
A couple of nits, but that's all. This is looking *so* much nicer.

(I'm really hoping to get back to my side of things tomorrow...)

File: standard/arrays.md (hunk 1 above)
Comment (RexJaeschke) [reply standard/arrays.md]: I agree to not use one.

Comment (jskeet) [reply standard/arrays.md]: Agreed - but we should add the braces and new lines.

**Review by RexJaeschke — APPROVED** [review]

**Pull request merged**